dbus_objects/integration/__init__.py

```diff
@@ -48,7 +48,8 @@ def __init__(
 
     @dbus_objects.object.dbus_method(return_names=('xml',))
     def introspect(self) -> str:  # noqa: C901
-        xml = ET.Element('node', {'xmlns:doc': 'http://www.freedesktop.org/dbus/1.0/doc.dtd'})
+        #xml = ET.Element('node', {'xmlns:doc': 'http://www.freedesktop.org/dbus/1.0/doc.dtd'}) # See: FFY00/dbus-objects#20.
```

Comment:
Great! Now you just need to put a space here (`# xml = ...`) to make flake8 happy :sweat_smile:

Reply:
Haha, the joys of helpful tools ;-p